src/tokenizer/index.js

```diff
@@ -8,6 +8,7 @@ import Token from './token';
 import KeyEvent from '../keyevent';
 import Typeahead from '../typeahead';
 import classNames from 'classnames';
+import objectAssign from "object-assign";
```

Comment:
It's a dependency for React. But maybe it should be added? It's better to be declarative.